sites/public/styles/overrides.scss

```diff
@@ -404,6 +404,11 @@ li.progress-nav__item {
   }
 }
 
+.accessibility-filter-selector {
+  column-count: 2;
```

Comment:
The checkboxes seem a little spread out, can you snug them up a bit to more closely match the mocks? In both the filter form and the eligibility questionnaire.

Reply:
Fixed!